src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Ball lotteryBall = new Ball();
+        lotteryBall.setColor(new ColorSupplier().getRandomColor());
+        lotteryBall.setNumber(new Random().nextInt(100));
```

Comment:
All magic numbers in your code should be constants. How do you think what max number will be returned by random.nextInt(100);?
Think about which variables should be local in the method and which should be class-level